docs/pbc/all/dynamic-multistore/202404.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,80 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
+|name        | v | string | DE | Define store name. |
+
+Configure a new store as described here: [Import configuration for store](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)
+
+
+When you add a new store, to enable store-related entities for customers, you need to assign them to the store. Some of the store related entities:
+- Products
+- Categories
+- CMS entities
+- Prices
+
+To avoid manually assigning entities in the Back Office, you can assign them using data import. For more details, see
+
+See: [Import minimum set of data for store](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/import-stores.html)
+
+
+## What is changed when Dynamic Multistore is enabled?
+
+Dynamic Multistore enables the following changes to the project:
+
+EU and US regions are used as an example.
+
+EU region has two stores: DE and AT. US region has one store: US.
+
+- URLs for Storefront, Back Office, Merchant Portal and Glue API contain region instead of store name. For example–the URL of the Back Office is `https://backoffice.eu.mysprykershop.com` instead of `https://backoffice.de.mysprykershop.com`.
+
+- RabbitMQ virtual hosts contain region instead of store. For example–`eu-docker` instead of `de-docker`.
+![rabbitmq-virtual-hosts-non-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/rabbitmq-virtual-hosts-non-dms.png)
+![rabbitmq-virtual-hosts-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/rabbitmq-virtual-hosts-dms.png)
+
+- Jenkins job names contain region instead of store. For example–`EU_queue-worker-start` instead of `DE_queue-worker-start`.
+![jenkins-jobs-non-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/jenkins-jobs-non-dms.png)
+![jenkins-jobs-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/jenkins-jobs-dms.png)
+
+- Elasticsearch indexes contain store as a part of the index name for Dynamic Multistore enabled and disabled modes.
+
+- Redis keys contain store as a part of the key name for Dynamic Multistore enabled and disabled modes.
+
+- When Dynamic Multistore is enabled, customers can switch between available stores for a region. When a customer changes a store, it's set to the `_store` query parameter. Using the query parameter, the store is added to session under the `current_store` key. It's used for fetching store-related data.
+
+![yves-store-switcher](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/yves-store-switcher.png)
+
+## Performance
+
+The number of stores affects data import speed: the more stores you have, the slower data import is.
+
+Take into account data import request body limit when you import data for many stores.
```

Comment:
How do I know this limit or what do I do with this information?

Reply:
Removed

Reply:
Removed once more )